src/utils.ts

```diff
@@ -394,7 +394,11 @@ export const setMaxStalePeriod = async (
     FORKED_NETWORK == "ethereum" ||
     FORKED_NETWORK == "sepolia" ||
     FORKED_NETWORK == "opbnbmainnet" ||
-    FORKED_NETWORK == "opbnbtestnet"
+    FORKED_NETWORK == "opbnbtestnet" ||
+    FORKED_NETWORK == "opmainnet" ||
+    FORKED_NETWORK == "opsepolia" ||
+    FORKED_NETWORK == "zksyncmainnet" ||
+    FORKED_NETWORK == "zksyncsepolia"
```

Comment:
We should add base network here as well, IMO.

Reply:
https://github.com/VenusProtocol/vips/pull/424/commits/ac889b152cd2358304c20c252882b66a8429c66f